src/main/resources/instructorEmailFragment-courseJoin.html

```diff
@@ -1,14 +1,14 @@
 <p>
-    The course <em>${courseName}</em> is using the TEAMMATES System to
-    collect peer evaluations. You have been assigned an instructor role
-    for the course by ${inviterName}, who can be reached at ${inviterEmail}.
+  The course <em>${courseName}</em> is using the TEAMMATES System to
+  collect peer evaluations. You have been assigned an instructor role
+  for the course by ${inviterName}, who can be reached at ${inviterEmail}.
```

Comment:
Let's improve the sentence breaks. The previous one looks like they're auto-formatted by some IDE with preset line length limit.
```html
The course ... collect peer evaluations.
You have been assigned ... ${inviterEmail}.
```